packages/core/src/components/chip/chip.scss

```
@@ -125,6 +121,13 @@ $outline-border-width: 0.0625rem;
 
     .slot-container {
       @include text-truncation.ellipsis;
+      flex: 0 1 auto;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)